.github/workflows/publish.yml

```diff
@@ -7,8 +7,6 @@ on:
 
 jobs:
   call-publish-workflow:
-    uses: deanmarcussen/NuGet-Publishing-GitHub-Actions/.github/workflows/publish.yml@v1
-    with:
-      source: https://nuget.cloudsmith.io/lombiq/Helpful-Libraries/v3/index.json
+    uses: Lombiq/NuGet-Publishing-GitHub-Actions/.github/workflows/publish.yml@v1
     secrets:
-      apikey: ${{ secrets.LOMBIQ_CLOUDSMITH_API_KEY }}
+      apikey: ${{ secrets.DEFAULT_PUBLISH_API_KEY }}

```

Comment:
Let's call it `LOMBIQ_NUGET_PUBLISH_API_KEY`.

Reply:
Done